docs/querying/sql-functions.md

```diff
@@ -116,11 +116,29 @@ Counts distinct values of a string, numeric, or `hyperUnique` column using Druid
 
 ## APPROX_COUNT_DISTINCT_DS_HLL
 
-`APPROX_COUNT_DISTINCT_DS_HLL(expr, [<NUMERIC>, <CHARACTER>])`
+Approximates distinct values of an HLL sketch column or a regular column. The [HLL sketch documentation](../development/extensions-core/datasketches-hll.md) describes the optional `lgK` and `tgtHllType` arguments.
 
-**Function type:** [Aggregation](sql-aggregations.md)
+* **Syntax:** `APPROX_COUNT_DISTINCT_DS_HLL(expr, [lgK, tgtHllType])`
+* **Function type:** Aggregation
+
+<details><summary>Example</summary>
+
+The following example returns the approximate number of distinct tail numbers in the `flight_carriers` datasource.
+
+```sql
+SELECT APPROX_COUNT_DISTINCT_DS_HLL("Tail_Number") AS "estimate"
+FROM "flight-carriers"
+```
+
+Returns the following:
+
+| `estimate` |
+| -- |
+| `4686` |
```

Comment:
I got 4,686

Reply:
Discussed with Charles on another PR, visibly it returns `4,686`. But if you select show full value it shows `4686`. Charles was not against the format without a comma